Source/MediaInfo/Multiple/File_P2_Clip.h

```diff
@@ -47,7 +46,7 @@ private :
 
     //Buffer - File header
     bool FileHeader_Begin();
-

```

Comment:
For next time: try to avoid unecessary diff e.g. empty line removal (here and above too).